core/src/main/scala/com/databricks/labs/remorph/coverage/ReportEntry.scala

```diff
@@ -11,16 +13,24 @@ case class ReportEntryHeader(
 case class ReportEntryReport(
     parsed: Int = 0, // 1 for success, 0 for failure
     statements: Int = 0, // number of statements parsed
-    parsing_error: Option[String] = None,
+    parsing_error: Option[RemorphError] = None,
```

Comment:
In my changes, I had to make this a List[RemorphError] as the parser can return more than one error message. Unless you have used some magic here?

Unfortunately we have both decided to change this and error capture in general,  at the same time. Fortunately we have arrived at more or less the same solution.

Reply:
Yeah, going for `List[RemorphError]` here is something I considered as well. Not only here, but in `Failure` as well, since we ultimately want the ability to accumulate multiple errors. But I deemed that to a future PR.

Another way would be to have something of the form 
```scala
case class RemorphErrors(errors: Seq[RemorphError]) extends RemorphError
```
To implement such accumulation while minimizing the amount of changes.